test/loopback.test.js

```diff
@@ -608,4 +611,96 @@ describe('loopback', function() {
       expect(methodNames).to.include('prototype.instanceMethod');
     });
   });
+
+  describe('Remote method inheritance', function() {
+    function getAllMethodNamesWithoutClassName(Model) {
+      return Model.sharedClass.methods().map(function(m) {
+        return m.stringName.replace(/^[^.]+\./, ''); // drop the class name
+      });
+    }
+    var app;
+
+    beforeEach(function() {
+      app = loopback({ localRegistry: true });
+    });
+
+    it('create model inherits method from base model', function() {
```

Comment:
In the light of https://github.com/strongloop/loopback/pull/2703/files#r77774974, I am proposing

```js
it('inherits remote methods defined via createModel`)
```